main.go

```diff
@@ -3,11 +3,89 @@ package main
 import (
 	"github.com/gin-gonic/gin"
 	"net/http"
+	"log"
+	//"database/sql"
+	
+	_ "github.com/lib/pq"
+		"github.com/jmoiron/sqlx"
 )
 
-func main() {
-	router := gin.Default()
-	
+var router = gin.Default()
```

Comment:
Initialize this in `initAPI()` method instead. This should be just variable declaration.